background.js

```diff
@@ -0,0 +1,471 @@
+'use strict';
+
+/* Enabled listeners that catch start of test run and Azure DevOps session that belongs to test run. */
+let initialListeners = [];
+/* Enabled listeners that catch update events of a test run. */
+let listeners = [];
+
+/* Map Chrome Tab Id to: URL of Test Run in Azure DevOps, subject to be displayed in Teamscale's Test Run View. */
+let tabToAdosTestRunUrlMap = {};
+/* Map Chrome Tab Id to: Test Case Id (not Test Run) as identified in Azure DevOps. */
+let tabToTestCaseMap = {};
+/* Map Test Case Id to: Parameters of the currently executed test run. Parameters are only sent on first update. */
+let testParametersMap = {};
+
+/* Map Chrome Tab Id to: Azure DevOps Session. This information currently unused. */
+let tabToSessionMap = {};
+/* Map Azure DevOps Session to: Azure DevOps User. This information is currently unused. */
+let sessionToUserMap = {};
+
+/* Options of the extension. Loaded once from the Chrome Storage. */
+let configOptions = {};
+
+/* Cookie of the active Teamscale session, needed for authenticated requests to Teamscale. */
+let teamscaleSession;
+
+/* Storage of all generated log messages (i.e. Test Events and Report Status Log Message from Teamscale. */
+let logMessages = [];
+
+const tsSapCallFinishedMsg = 'call finished';
+
+const tsServerOptionName = 'ts-server';
+const tsProjectOptionName = 'ts-project';
+const sapUserOptionName = 'sap-username';
+const extendedUriFilterOptionName = 'extended-uri-filter';
+const allOptions = [tsServerOptionName, tsProjectOptionName, sapUserOptionName, extendedUriFilterOptionName];
+
+const API_ON_PREMISE_CALL_OPEN_TEST_RUNNER = '/_api/_wit/pageWorkItems?__v=5';
+const API_SERVICES_CALL_OPEN_TEST_RUNNER = '/_apis/Contribution/dataProviders/query';
+const API_CALL_UPDATE_TEST_RUN_SUFFIX = '/_api/_testresult/Update?teamId=&__v=5';
+
+const standardUriFilter = ["https://*.visualstudio.com/*", "https://dev.azure.com/*"];
```

Comment:
```suggestion
const standardUriFilter = ['https://*.visualstudio.com/*', 'https://dev.azure.com/*'];
```
to adhere to our JS/TS coding guideline?

Reply:
done